org.getmonero.Monero.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: org.getmonero.Monero
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --share=ipc
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: monero-wallet-gui
+modules:
+  - name: monero-bin
+    buildsystem: simple
+    sources:
+    - type: archive
+      url: https://downloads.getmonero.org/gui/monero-gui-linux-x64-v0.17.2.0.tar.bz2
```

Comment:
Wouldn't be better to build this one from sources? I think following the `Dockerfile` for Linux could be a good start.